src/workerd/api/streams/internal.h

```diff
@@ -270,6 +273,8 @@ class WritableStreamInternalController: public WritableStreamController {
 
   void visitForGc(jsg::GcVisitor& visitor) override;
 
+  void setHighWaterMark(uint64_t highWaterMark);
```

Comment:
note: this method currently is not used here but added for later testing. I'm fine with removing it for now tho if folks prefer.